Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -696,7 +695,7 @@ bool VolumeVerifier::ShouldHaveChannelPartition() const
       "RFNE01", "RFNJ01", "RFNK01", "RFNP01", "RFNW01", "RFPE01", "RFPJ01", "RFPK01", "RFPP01",
       "RFPW01", "RGWE41", "RGWJ41", "RGWP41", "RGWX41", "RMCE01", "RMCJ01", "RMCK01", "RMCP01",
   };
-  assert(std::is_sorted(channel_discs.cbegin(), channel_discs.cend()));
+  DEBUG_ASSERT(std::is_sorted(channel_discs.cbegin(), channel_discs.cend()));
```

Comment:
This (along with `ShouldBeDualLayer`) is the only place where I used `DEBUG_ASSERT`, as this isn't an assertion that needs to be run in most cases.  In C++20, `is_sorted` becomes `constexpr`, so it could be replaced with `static_assert`.

Reply:
Done by https://github.com/dolphin-emu/dolphin/pull/13084.